test/files/neg/t12851b/C_2.scala

```diff
@@ -0,0 +1,2 @@
+// scalac: -Werror
+class C extends T1 with T2
```

Comment:
`T1` and `T2` could be defined on the user's classpath so he might not be able to address the warning at the source. But the compiler can't know, so I think it's good to keep the warning.

Reply:
It may not be the source's problem. The traits could be from different libraries. It's the fault of the mixer, who got mixed up.